app/helpers/meta_helper.rb

```diff
@@ -1,5 +1,6 @@
 module MetaHelper
   def open_graph_tags
-    content_for :open_graph_tags
+    render(OpenGraphTagsComponent.new) unless content_for?(:open_graph_tags)
+    content_for(:open_graph_tags)
```

Comment:
Following from your Twitter thread here: https://twitter.com/joemasilotti/status/1497341331315650560?s=20&t=nc7y6ITf4HnEDPGLhai9LA.

How about:

```
render(OpenGraphTagsComponent.new, tags: content_for(:open_graph_tags)
```

Then in the component itself, return the tags if they're present, otherwise render the component's view.

Reply:
Maybe I'm missing something, but doesn't this mean I have to add back in the following block every time I render the component?

```
<% content_for(:open_graph_tags) do %>
  <%= render OpenGraphComponent.new(...) %>
<% end %>
```